coordinates/sites.json

```diff
@@ -872,5 +872,130 @@
         "longitude": 10.504496611198473,
         "longitude_unit": "degree",
         "source": "LALSuite: https://git.ligo.org/lscsoft/lalsuite/-/blob/lalsuite-v6.70/lal/lib/tools/LALDetectors.h"
-    }
+    },
+	"SuperK": {
+        "source": "SK detector paper, Fukuda et al. 2003",
+        "name": "Super-Kamiokande",
+        "elevation": 370,
+        "elevation_unit": "meter",
+        "latitude": 36.425722,
+        "latitude_unit": "degree",
+        "longitude": 137.310306,
+        "longitude_unit": "degree",
+        "timezone": "Japan Standard Time",
+        "aliases": [
+            "superk",
+	    "Superk",
+	    "superK"
+        ]
+    },
+	"HyperK": {
+        "source": "Design Report, arXiv:1805.04163v2",
+        "name": "Hyper-Kamiokande",
+        "elevation": 514,
+        "elevation_unit": "meter",
+        "latitude": 36.355583,
+        "latitude_unit": "degree",
+        "longitude": 137.313639,
+        "longitude_unit": "degree",
+        "timezone": "Japan Standard Time",
+        "aliases": [
+            "hyperk",
+	    "Hyperk",
+	    "hyperK"
+        ]
+    },
+	"ARCA": {
+        "source": "SNEWS 2.0 Collaboration",
```

Comment:
For KM3NeT, these numbers are published in their [2016 Letter of Intent](https://doi.org/10.1088/0954-3899/43/8/084001). Section 2.1 has numbers for ARCA (“The KM3NeT-Italy infrastructure is located at 36º 16' N 16º 06' E at a depth of 3500 m”) and 2.2 has numbers for ORCA (“The KM3NeT-France infrastructure is located at 42º 48' N 06º 02' E at a depth of 2450 m”).

@amgraf If you haven’t received any responses from the respective collaborators, it might be quicker to look for the main detector paper (or design report) of each experiment on InspireHEP (in case you don’t know about it, the [`collaboration:XXX`](https://inspirehep.net/literature?sort=mostcited&size=25&page=1&q=collaboration%3Asno) search syntax + sort by citations is very useful!); usually that will contain location data.